fantasy-esports-app/src/app/components/updateProfilePic.tsx

```diff
@@ -0,0 +1,16 @@
+function UpdateProfilePic() {
```

Comment:
Could you rename the file to update_profile_picture.tsx

Reply:
Mommy and Daddy (Me and Cesar) are debating file naming so ignore this for the meantime